database/README.md

```diff
@@ -251,7 +251,7 @@ The `FirebaseRecyclerPagingAdapter` is built on top of the [Android Paging Suppo
 Before using the adapter in your application, you must add a dependency on the support library:
 
 ```groovy
-implementation 'android.arch.paging:runtime:1.x.x'
+implementation 'androidx.paging:paging-runtime:2.x.x'
```

Comment:
Is the part of the paging library (`PagedList.Config`) that we use unchanged from 1.x to 2.x?

Reply:
@samtstern Yes, `PagedList.Config` remains unchanged.